### ISSUE TITLE
不向全局事务发起方的undolog表写数据,应该怎么查找原因

### ISSUE THREAD

**Issue by MissHuZeQuan** [seata/seata-samples#260]

**Comment by slievrly** [comment]
@MissHuZeQuan 每个微服务节点向拥有的自有业务库中undo_log表写数据，不是集中写到发起方，在抛出异常处打个断点观察undo_log表数据。

**Comment by MissHuZeQuan** [comment]
是的,每个微服务节点向拥有的自有业务库中undo_log表写数据，不是集中写到发起方,这个我知道,关键是没有抛出异常,也没向undolog表写入数据

**Comment by seustones** [comment]
同遇到，打断点发现注解GlobalTransactionalInterceptor并没有进去，使用老一点版本的seata打包的客户端测试时是可以正确执行的，新版本打包后测试遇到这个问题，很奇怪，求大佬解答

**Comment by seustones** [comment]
[https://github.com/seata/seata/pull/1858](url) 已fix

**Comment by seustones** [comment]
[#1858](https://github.com/seata/seata/pull/1858) 已fix
